local/mymedia/mymedia.php

```diff
@@ -54,8 +54,6 @@
     'height' => '600px',
     'width' => '100%',
     'allowfullscreen' => 'true',
-    'webkitallowfullscreen' => 'true',
```

Comment:
No reason other than that.
Anyway, Moodle's basic browser requirements are:
IE: 9+, Chrome: 30+, Firefox: 25+, Safari: 6+.
All of these browsers, except for Safari 6 (support added in 7.x), support the 'allowfullscreen'.

And Safari 6.x has a 0.02% usage worldwide...
Obviously I have no problem putting back the `webkitallowfullscreen`attribute, costs me nothing :)